docs/modules/deployment/pages/system-scalability/time-series-storage/newts.adoc

```diff
@@ -90,6 +90,6 @@ describe table samples;
 systemctl restart opennms
 ----
 
-With this infomration set, {page-component-title} will begin persisting metrics using the Newts engine when restarted.
+With this information set, {page-component-title} will begin persisting metrics using the Newts engine when restarted.
```

Comment:
Fixed in https://github.com/OpenNMS/opennms/pull/3702

Reply:
I've removed this change from the PR.